meinberlin/assets/scss/components_user_facing/_project-tile.scss

```diff
@@ -13,20 +13,33 @@ a.project-tile {
         height: 100%;
     }
 
-    &--horizontal {
+    &--horizontal:not(.project-tile--map) {
         border-bottom: 1px solid $gray-lighter;
         padding-bottom: 1.5em;
         padding-top: 1.5em;
     }
 }
 
 .project-tile__image-wrapper {
-    flex: 1
+    flex: 1;
+    min-width: 0;
+
+    .project-tile--map & {
+        flex: 0 0 auto;
+
+        @media screen and (min-width: $breakpoint-tablet) {
+            flex: 3;
+        }
+    }
 }
 
 .project-tile__image {
     object-fit: cover;
-    width: 100%;
+    max-width: 100%;
+    height: auto;
+    max-height: 100%;
+    width: auto;
```

Comment:
width:auto might be causing the issue with the images

Reply:
this is what it looks like without
![tiles-aspectratio](https://github.com/user-attachments/assets/79476415-ae39-49a6-8631-94fa5896390c)